src/lib/loaders/__tests__/api.test.js

```diff
@@ -43,15 +41,14 @@ describe("API loaders", () => {
       })
     })
 
-    it("caches the response for the lifetime of the loader", () => {
-      return Promise.all([loader(), loader()]).then(responses => {
+    it("caches the response for the lifetime of the loader", () =>
```

Comment:
Here's another good example about taking brackets away -- if someone wanted to log or create a variable before running promise.all they would have to adjust the structure of the block before doing so, and add in two addition brackets at the top and the bottom.

Reply:
I definitely understand your preference. I tend to find myself on the flip-side of that opinion, where implicit returns are shorter and more readable, and there's an easy-at-a-glance way to tell if a function is just returning something or if it's doing more work than that. 

Implicitly returning functions carry a certain semantic value that allows me to offload an amount of cognitive load when reading code. That matters a lot to me, but it's just a preference.

I'm happy to be overruled if we have a company policy around this, but don't necessarily feel the need to privilege one preference over another, right?